docs/anomaly-detection/QuickStart.md

```diff
@@ -0,0 +1,115 @@
+and [Overview](https://docs.victoriametrics.com/anomaly-detection/overview.html) of how `vmanomaly` service works.
+
+## How to install and run `vmanomaly`
+
+> To run `vmanomaly` you need to have VictoriaMetrics Enterprise license. You can get a trial license key [**here**](https://victoriametrics.com/products/enterprise/trial/index.html).
+
+The following options are available:
+
+- [To run Docker image](#docker-image)
+- [To run in Kubernetes with Helm charts](#helm-charts)
+
+
+### Docker image
+> To run `vmanomaly` you need to have a VictoriaMetrics Enterprise [licence](https://victoriametrics.com/products/enterprise/) or request a trial [here](https://victoriametrics.com/products/enterprise/trial/index.html).
+
+Below are the steps to get `vmanomaly` up and running inside a Docker container:
+
+1. Pull Docker image:
+
+```sh
+docker pull victoriametrics/vmanomaly:latest
+```
+
+2. (Optional step) tag the `vmanomaly` Docker image:
+
+```sh
+docker image tag victoriametrics/vmanomaly:latest vmanomaly
+```
+
+3. Start the `vmanomaly` Docker container with a *license file*, use the command below.
+**Make sure to replace `YOUR_LICENSE_FILE_PATH`, and `YOUR_CONFIG_FILE_PATH` with your specific details**:
+
+```sh
+export YOUR_LICENSE_FILE_PATH=path/to/license/file
+export YOUR_CONFIG_FILE_PATH=path/to/config/file
+docker run -it -v $YOUR_LICENSE_FILE_PATH:/license \
+               -v $YOUR_CONFIG_FILE_PATH:/config.yml \
+               vmanomaly /config.yml \
+               --license-file=/license
+```
+
+See also:
+
+- You can verify licence online and offline. See the details [here](https://docs.victoriametrics.com/anomaly-detection/overview/#licensing).
+- [How to configure `vmanomaly`](#how-to-configure-vmanomaly)
+
+### Kubernetes with Helm charts
+
+You can run `vmanomaly` in Kubernetes environment
+with [these Helm charts](https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-anomaly/README.md).
+
+
+## How to configure vmanomaly
+To run `vmanomaly` you need to set up configuration file in `yaml` format.
+
+Here is an example of config file that will run [Facebook Prophet](https://facebook.github.io/prophet/) model, that will be retrained every 2 hours on 14 days of previous data. It will generate inference (including `anomaly_score` metric) every 1 minute.
+
+
+```yaml
+scheduler:
+  infer_every: "1m"
+  fit_every: "2h"
+  fit_window: "14d"
+
+models:
+  prophet_model:
+    class: "model.prophet.ProphetModel"
+    args:
+      interval_width: 0.98
+
+reader:
+  datasource_url: "http://victoriametrics:8428/" # [YOUR_DATASOURCE_URL]
+  sampling_period: "1m"
+  queries: 
+    # define your queries with MetricsQL - https://docs.victoriametrics.com/metricsql/
+    cache: "sum(rate(vm_cache_entries))"
+
+writer:
+  datasource_url:  "http://victoriametrics:8428/" # [YOUR_DATASOURCE_URL]
+```
+
+
+Next steps:
```

Comment:
@tenmozes changed Next steps

Reply:
please simplify

Reply:
I'm not sure if I understand correctly which part to simplify. 
I changed to 

```Define how often to run and make inferences in the [scheduler](https://docs.victoriametrics.com/anomaly-detection/components/scheduler/) section of a config file.```

@tenmozes is that it?